prdoc/pr_9461.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+
+title: 'Expose migrating keys'
+doc:
+  - audience: Runtime Dev & Node Dev
```

Comment:
Not sure if this is valid YAML, you can add the schema to VsCode if you use that https://github.com/paritytech/prdoc?tab=readme-ov-file#pr-doc  
Or run the CLI `prdoc check`.